test-series/1/test-1.1.0.ipynb

```diff
@@ -0,0 +1,1263 @@
+{
```

Comment:
<p>Bobby Pascua commented on PR 51 what this frequency-dependent scaling is. It should probably be written in here as well.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>This has been done.</p>